requirements.txt

```diff
@@ -0,0 +1,4 @@
+torch >= 2.1.0
```

Comment:
we maybe wanna use the official wheel for pytorch (and torchvision/audio) which is distributed on pytorch's official website.
there's some info at https://github.com/pytorch/pytorch/issues/29745#issuecomment-553588171 and/or https://stackoverflow.com/questions/60912744/install-pytorch-from-requirements-txt

Reply:
also closing this